### PR TITLE
More Hacks

### PR DESCRIPTION
**Pull request by FuxorLuck**

### CODE REVIEW

File: cheats_ni/C293DD66.pnach [hunk 1]
```diff
@@ -1,3 +1,6 @@
 //No interlacing
 patch=1,EE,201E1AB0,word,30420000
 patch=1,EE,00101f5c,word,00000000 //64420008
+
```
Comment (Mrlinkwii) [inline cheats_ni/C293DD66.pnach:4]: which broke effect ?

File: cheats_ni/38145680.pnach [hunk 2]
```diff
@@ -0,0 +1,6 @@
+gametitle=Shadow Tower Abyss (English patch)
```
Comment (Mrlinkwii) [inline cheats_ni/38145680.pnach:1]: dose this apply to the non patched

Comment (Mrlinkwii) [inline cheats_ni/38145680.pnach:1]: dose this apply to the non patched?

File: cheats_ws/F3BDB2E6.pnach [hunk 3]
```diff
@@ -11,4 +11,12 @@ patch=1,EE,00202c98,word,44810000
 patch=1,EE,00202ca0,word,4600c602
 
 // Render-Fix
-patch=1,EE,0022531c,word,3c033fc0
\ No newline at end of file
+patch=1,EE,0022531c,word,3c033fc0
+
+
+//16:9 Text box
```
[the remark this answers is not in the document]
Comment (refractionpcsx2) [reply cheats_ws/F3BDB2E6.pnach:17]: It says why it's commented out.

File: cheats_ws/CDD36744.pnach [hunk 4]
```diff
@@ -0,0 +1,12 @@
+gametitle=Mystic Nights (English Patched)
```
[the remark this answers is not in the document]
Comment (FuxorLuck) [reply cheats_ws/CDD36744.pnach:1]: yes

**Pull request merged**